runtime/nls/shrc/j9shr.nls

```diff
@@ -2540,12 +2540,12 @@ J9NLS_SHRC_OSCACHE_ERROR_INIT_SEMAPHORE_POST_SEM_USERLOCK_MUTEX.system_action=Th
 J9NLS_SHRC_OSCACHE_ERROR_INIT_SEMAPHORE_POST_SEM_USERLOCK_MUTEX.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
 
-J9NLS_SHRC_CC_AQUIRE_LOCK_FAILED_ENTER_MUTEX=SH_OSCachesysv::acquireWriteLock() call to j9shsem_wait has failed with error %d.
+J9NLS_SHRC_CC_ACQUIRE_LOCK_FAILED_ENTER_MUTEX=SH_OSCachesysv::acquireWriteLock() call to j9shsem_wait has failed with error %d.
```

Comment:
All the j9shr*.nls files need to have the same change.

Reply:
https://github.com/eclipse/openj9/pull/5949/commits/e86b627b6700edab589c44a97fe5901e16cc61b3